pandas/core/frame.py

```diff
@@ -1744,6 +1744,8 @@ def memory_usage(self, index=False, deep=False):
                              index=['Index']).append(result)
         return result
 
+    __sizeof__ = com._sizeof
```

Comment:
and FYI you don't need to define this in sub-classes (you already defined this in ``base``). What I mean here is remove ALL definition of ``__sizeof__`` except the one on ``PandasObject``, and handle the case where the object doesn't have ``.memory_usage`` defined.